hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -62,6 +62,18 @@
     public static final String PROP_VERSION_CHECK_ENABLED = "hazelcast.version.check.enabled";
     public static final String PROP_PREFER_IPv4_STACK = "hazelcast.prefer.ipv4.stack";
     public static final String PROP_IO_THREAD_COUNT = "hazelcast.io.thread.count";
+
+    /**
+     * The interval in seconds between {@link com.hazelcast.nio.tcp.handlermigration.IOBalancer IOBalancer}
+     * executions. The shorter intervals will catch I/O Imbalance faster, but they will cause higher overhead.
+     *
+     * Please see documentation of {@link com.hazelcast.nio.tcp.handlermigration.IOBalancer IOBalancer} for
+     * detailed explanation of the problem.
+     *
+     * Default value is 20 seconds. A negative value disables the balancer.
```

Comment:
0 simply means there will be no sleep between executions. 
It doesn't make much sense, but it's not really too different from e.g. 1.